### ISSUE TITLE
The test accuracy using the pre-training model is lower than that in the paper

### ISSUE THREAD

**Issue by volare1996** [LiyaoTang/contrastBoundary#6]
Using pre-training model
-------------------------------------------------------------------------------------------
OA=87.06 
mACC=68.31  | 94.33 99.39 93.51 0.00  32.67 44.93 61.39 92.20 86.43 80.77 69.41 58.78 74.29
mIoU=61.29  | 92.18 98.25 78.77 0.00  29.74 40.42 51.87 84.59 74.31 69.00 67.60 53.74 56.34
	 -- bound IoU --
	0.1	 - 0.5751144616243722

Paper
==> Confusion on full clouds - final:
---------------------------------------------------------------------------------------------------------------
mIoU=69.36 OA=90.64 mACC=75.22  | 93.86 98.41 84.19 0.00  37.00 57.65 71.87 91.70 81.75 77.75 75.57 69.05 62.87
	 -- bound IoU --
	0.1	 - 0.6145995694137792

**Comment by LiyaoTang** [comment]
Hi, could you upload the full log here? I'm wondering if there is a mismatch between the snapshot and the selected model config.
